goagen/gen_client/generator.go

```diff
@@ -334,13 +333,50 @@ func (g *Generator) generateResourceClient(pkgDir string, res *design.ResourceDe
 				}
 			}
 		}
+		initParams := func(att *design.AttributeDefinition) []*paramData {
```

Comment:
Looks like the same logic as the one used in `generateActionClient` - maybe it's time to make this its own function and call it from both places?